third-party/realdds/include/realdds/dds-device-server.h

```diff
@@ -14,11 +14,9 @@ namespace realdds {
 
 // Forward declaration
 namespace topics {
+class notification;

```

Comment:
Can you explain when is a topic "raw"?
Why is `device_info` raw but `notification` not?

Reply:
This is the non-raw notification you're looking at. There's a `raw::notification`, too